src/commercetools/testing/custom_objects.py

```diff
@@ -79,6 +78,16 @@ def urls(self):
             ("^$", "GET", self.query),
             ("^$", "POST", self.create),
             ("^(?P<container>[^/]+)/(?P<key>[^/]+)$", "GET", self.get_by_container_key),
+            (
+                "^(?P<container>[^/]+)/(?P<key>[^/]+)$",
+                "POST",
+                self.get_by_container_key,
+            ),
+            (
+                "^(?P<container>[^/]+)/(?P<key>[^/]+)$",
+                "DELETE",
+                self.get_by_container_key,
+            ),
```

Comment:
Should this be?
```suggestion
            (
                "^(?P<container>[^/]+)/(?P<key>[^/]+)$",
                "POST",
                self.update_by_container_key,
            ),
            (
                "^(?P<container>[^/]+)/(?P<key>[^/]+)$",
                "DELETE",
                self.delete_by_container_key,
            ),
```

Or something similar?

Reply:
get_by_container just checks if the container exists and if so, it creates a Commercetools response. Creating individual methods for this would result in redundant code.